tests/pages/api/apiUtils/userData.test.ts

```diff
@@ -91,10 +92,35 @@ describe('getSingleTicketJson', () => {
                     startDate: '2020-12-17T09:30:46.0Z',
                     endDate: '2020-12-18T09:30:46.0Z',
                 },
+                [FULL_TIME_RESTRICTIONS_ATTRIBUTE]: {
```

Comment:
Can this object be pulled out into a variable since it seems to be used in a few places?